doc/api/util.md

```diff
@@ -580,8 +575,8 @@ console.log(inspect(weakSet, { showHidden: true }));
 // WeakSet { { a: 1 }, { b: 2 } }
 ```
 
-The `sorted` option makes sure the output is identical, no matter of the
-properties insertion order:
+The `sorted` option ensures that object insertion order does not impact the
```

Comment:
I am not sure if `object insertion order` is something everyone will understand (but neither the former description). What about:

```
The `sorted` option ensures that the object properties insertion order does not
impact the result of `util.inspect()`.
```